### PR TITLE
Sprint 26

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,16567 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (kkoskelin) [inline yarn.lock:1]: Um, are we using Yarn? If not, this really ought to be removed.

**Review by kkoskelin — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (matthopson) [reply yarn.lock:1]: I feel like we're using this for `serverless-jetpack` - @sutt0n confirm?

Comment (sutt0n) [reply yarn.lock:1]: Can confirm...

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**